master/internal/db/postgres_cluster.go

```diff
@@ -131,3 +134,17 @@ func (db *PgDB) UpdateResourceAllocationAggregation() error {
 	}
 	return nil
 }
+
+// CheckIfRPUnbound checks to make sure both of the specified resource pools are not bound to any
+// workspaces and returns an error if they are.
+func (db *PgDB) CheckIfRPUnbound(defaultAux, defaultCompute string) error {
+	exists, err := Bun().NewSelect().Table("rp_workspace_bindings").
+		Where("pool_name IN (?)", bun.In([]string{defaultAux, defaultCompute})).
+		Exists(context.TODO())
+	if err != nil {
+		return err
+	} else if exists {
+		return fmt.Errorf("default resource pools can not be bound to any workspaces")
```

Comment:
Can we include the pool name that is the issue and also mention if it is the compute vs aux in the error message?

Stopping master from starting could be confusing.